dpctl/_sycl_device.pyx

```diff
@@ -395,6 +379,96 @@ cdef class SyclDevice(_SyclDevice):
                 "a SYCL filter selector string."
             )
 
+    @property
+    def aspect_host(self):
```

Comment:
Better to rename the properties as `has_aspect_xxx`

Reply:
This way it is clear that the return type must be a boolean